### PR TITLE
added onboarding flow (no upload photo section for now)

### LINKED ISSUE #16: Add onboarding flow

### PR DESCRIPTION
**Pull request by nayasinghania**
https://github.com/user-attachments/assets/833070d6-bd28-4802-bce0-d806ca97cd46

### CODE REVIEW

**Comment by ManasC478** [comment]
The components look amazing. Just some things to change.

for first page:
1. make the majors input a select instead of input because eventually you will fetch a list of majors from which the users can select
2. let's not add a 'other' options for status input. we don't need  year input and change 'status' to 'education level'
3. add input for grad_date. you can create your own component or use shadcn
4. let's add a interests input that can select multiple different interests. so they can select 'Web Dev', 'ML', 'Embedded' or more. The exact options are in the db_scripts/createdb.sql file as cs_fields_enum type at the top, in the server repo. you can make it a checkbox or dropdown checkbox.
5. let's remove 'team interest' input

for second page:
1. remove pronouns, bio, discord, and instagram inputs
2. see if you have fit the website, linkedin, github inputs to the first onboarding page. if it doesn't look good we can leave them on the second page

File: src/components/atoms/card/component.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export interface ICardProps extends React.ButtonHTMLAttributes<HTMLDivElement> {
```
Comment (ManasC478) [inline src/components/atoms/card/component.tsx]: checkout the comment i left in the discussion

**Review by ManasC478 — CHANGES_REQUESTED** [review]

**Comment by nayasinghania** [comment]
> The components look amazing. Just some things to change.
> 
> 1. make the majors input a select instead of input because eventually you will fetch a list of majors from which the users can select
> 2. let's not add a 'other' options for status input. we don't need  year input and change 'status' to 'education level'
> 3. add input for grad_date. you can create your own component or use shadcn
> 4. let's add a interests input that can select multiple different interests. so they can select 'Web Dev', 'ML', 'Embedded' or more. The exact options are in the db_scripts/createdb.sql file as cs_fields_enum type at the top, in the server repo. you can make it a checkbox or dropdown checkbox.
> 5. let's remove 'team interest' input

Ok, i'll get to work on making those changes now

**Comment by nayasinghania** [comment]
change to education level

status: required
major: required
grad date:required
purpose: optional
interests:optional
links:optional

**Review by ManasC478 — APPROVED** [review]

**Pull request merged**